tools/docker-compose/ansible/roles/sources/templates/nginx.locations.conf.j2

```diff
@@ -46,4 +46,5 @@ location @fallback {
     # Add trailing / if missing
     rewrite ^(.*)$http_host(.*[^/])$ $1$http_host$2/ permanent;
     proxy_pass http://runserver;
+    proxy_set_header Host $http_host;
```

Comment:
This is what we currently pass in the awx-operator nginx.conf as well:
* https://github.com/ansible/awx-operator/blob/devel/roles/installer/templates/configmaps/config.yaml.j2#L202-L203